src/includes/early-adopter-note.mdx

```diff
@@ -0,0 +1,6 @@
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
```

Comment:
```suggestion
If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily. [Link test](/product/). [External link test](https://sentry.io/for/performance/#updates-signup). [Mail link test](mailto:profiling@sentry.io).
```